### PR TITLE
Feature/staging indexd

### PR DESCRIPTION
**Pull request by bwalsh**
This PR adds fixes for gen3/helm and updates documentation for s3 buckets.

### CODE REVIEW

File: etl.yaml [hunk 1]
```diff
@@ -19,9 +19,13 @@ spec:
...
 
-    - name: host-mount
+    - name: studies-mount
       hostPath:
-        path: /Users/walsbr/aced/data_model/studies        
+        path: /Users/walsbr/aced/data_model/studies
+
+    - name: output-mount
+      hostPath:
+        path: /Users/walsbr/aced/data_model/output
```
Comment (lbeckman314) [inline etl.yaml:28]: I'll look for possible ways to make `hostPath` path mounts relative or otherwise host-agnostic.

**Review by lbeckman314 — APPROVED** [review]
Looks good! I'll start replicating the ETL steps locally.

File: etl.yaml [hunk 2]
```diff
@@ -84,6 +93,12 @@ spec:
           # mac silicon, build from scratch, avoids Postgresql SCRAM authentication problem
           pip install  psycopg2
 
+          # link back to mounted volumes
+          ln -s /studies /data_model/studies
+          ln -s /output /data_model/output
```
[the remark this answers is not in the document]
Comment (bwalsh) [reply etl.yaml:98]: @lbeckman314 Thanks.  I think I tried it.  However, it seemed like the git clone was messing it up.    Let's re-visit when we prepare a quay image for the etl pod.

**Comment by bwalsh** [comment]
@lbeckman314  merged to `feature/staging`

**Pull request merged**